dist/jquery.remodal.js

```diff
@@ -251,8 +251,10 @@
     remodal.$modal.append(remodal.$closeButton);
     remodal.$wrapper.append(remodal.$modal);
     remodal.$body.append(remodal.$wrapper);
-    remodal.$confirmButton = remodal.$modal.find('.' + namespace + '-confirm');
-    remodal.$cancelButton = remodal.$modal.find('.' + namespace + '-cancel');
+
+    remodal.$closeButton = remodal.$modal.find('[data-' + namespace + '-action="close"]');
+    remodal.$confirmButton = remodal.$modal.find('[data-' + namespace + '-action="confirm"]');
+    remodal.$cancelButton = remodal.$modal.find('[data-' + namespace + '-action="cancel"]');
```

Comment:
Definitely, you forgot to add this selectors here instead of `'.' + namespace + '-...'`:

https://github.com/Ashot-KR/Remodal/blob/actions-v0.7/dist/jquery.remodal.js#L266
https://github.com/Ashot-KR/Remodal/blob/actions-v0.7/dist/jquery.remodal.js#L273
https://github.com/Ashot-KR/Remodal/blob/actions-v0.7/dist/jquery.remodal.js#L284

And also, remove, please, this fields, they do not need anymore.